A32NX/ModelBehaviorDefs/Asobo/Airliner/Airbus.xml

```diff
@@ -1177,17 +1174,6 @@
 			<KNOB_NUM_STATE>7</KNOB_NUM_STATE>
```

Comment:
Can you make these temperature knobs have continuous mode instead of 7 states?

Reply:
I tried using the continuous knob template recycled from the light knobs but just couldn't get them to work for some reason. Maybe someone else will have better luck, or I'll get to them in another PR.